src/Routers/AppRouter.tsx

```diff
@@ -106,7 +105,7 @@ export default function AppRouter() {
         )}
         <main
           id="pages"
-          className="flex-1 overflow-y-auto bg-gray-100 focus:outline-none md:pb-2 md:pr-2"
+          className="flex-1 overflow-hidden bg-gray-100 focus:outline-none md:pb-2 md:pr-2"
```

Comment:
let's get other scroll issues fixed as well; there seems to be unnecessary scroll in many places.

<img width="1738" alt="image" src="https://github.com/user-attachments/assets/9cbdf3e5-c8f7-4519-a60e-bb71c792e094" />

<img width="1738" alt="image" src="https://github.com/user-attachments/assets/e316757b-8707-4d5e-bed1-cc2a1b3c3b97" />